README.md

```diff
@@ -1 +1,359 @@
+```json
+{
+    "pattern": "^s3/(?P<accession>.*)$",
+    "path": "https://download.gdi.nbis.se/s3/{accession}"
+},
+```
+
+Then change the port where the service is mapped to the port you want to expose (currently set to 8081) by changing the `docker-compose-htsget.yml` file (a sample can be found under `htsget` in this repo) as follows:
+```yaml
+services:
+  server:
+    ports:
+      - 8081:3000
+```
+
+## Deploying REMS
+Edit the configuration file `config.edn`, add/modify values for public_url, database_url, oidc-metadata-url, oidc-scopes, oidc-client-id and oidc-client-secret
+```edn
+:public-url "https://rems.gdi.nbis.se/"
+:database-url "postgresql://db:5432/rems?user=<REMS_DATABASE_USER>&password=<REMS_DATABASE_PASSWORD>"
+:oidc-metadata-url "https://login.elixir-czech.org/oidc/.well-known/openid-configuration"
+:oidc-scopes "openid profile email ga4gh_passport_v1"
+```
+
+The values for oidc-client-id and oidc-client-secret can be obtained from spreg, in our case from `https://services.aai.lifescience-ri.eu/spreg/auth/facilities/detail/<YOUR_PROJECT_ID>`
+
+To configure TLS for REMS, one needs to add three parameters, namely ssl-port, ssl-keystore and ssl-keystore-password. The value of ssl-port should be set as 3000, consequently, port for non TLS can be modified to e.g. 3001.
+
+The keystore file can be converted from certificate files generated by CertBot by the following commands, assuming the public key file is `./certs/cert.pem`, the private key file is `./certs/privkey.pem` and the CA file is `./certs/fullchain.pem`.
+
+From outside the container (on the VM) and under the root folder of the repo run
+```bash
+openssl pkcs12 -export \
+    -in ./certs/fullchain.pem \
+    -inkey ./certs/privkey.pem \
+    -out ./certs/rems.gdi.nbis.se.p12 \
+    -name rems.gdi.nbis.se \
+    -password "<CERTIFICATE_PASSWORD>"
+```
+
+Then run the following command from inside the rems container:
+```bash
+keytool -importkeystore \
+    -deststorepass <CERTIFICATE_PASSWORD> \
+    -destkeypass <CERTIFICATE_PASSWORD> \
+    -deststoretype pkcs12 \
+    -srckeystore rems.gdi.nbis.se.p12 \
+    -srcstoretype PKCS12 \
+    -srcstorepass <CERTIFICATE_PASSWORD> \
+    -destkeystore rems.gdi.nbis.se.keystore \
+    -alias rems.gdi.nbis.se
+```
+
+For the docker setup, add/modify ports of the service “app”
+```yaml
+ports:
+- "3000:3000"
+- "3001:3001"
+```
+
+Modify ports of the service “db” to 5433 to avoid conflict with another db container 
+```yaml
+ports:
+- "127.0.0.1:5433:5432"
+```
+
+Add the volume starter-kit-storage-and-interfaces_shared
+```yaml
+starter-kit-storage-and-interfaces_shared:
+  external: true
+```
+
+The `private-key.jwk` created when deploying REMS does not contain the “kid” field, therefore (for now) that needs to be added manually, by copying the same value from the `public-key.jwk`. This might have been fixed on the key creation script, in later REMS versions.
```

Comment:
```suggestion
The `private-key.jwk` created when deploying REMS does not contain the `kid` field, therefore (for now) that needs to be added manually, by copying the same value from the `public-key.jwk`. This might have been fixed on the key creation script, in later REMS versions.
```